frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -106,6 +107,7 @@ export const createBuildConfig = (
   formData: GitImportFormData,
```

Comment:
I would suggest we use a more explicit name here `createOrUpdateBuildConfig` and add a `verb` prop to say what we want to do `create` or `update`.